app/src/main/java/com/android/ootd/ui/feed/FeedScreen.kt

```diff
@@ -172,8 +176,15 @@ fun FeedList(
       val isLiked = likes[post.postUID] ?: false
       val count = likeCounts[post.postUID] ?: 0
 
+      // Get user data for this post owner, fallback to post data if missing
+      val userData = userDataMap[post.ownerId]
+      val username = userData?.username ?: post.name
+      val profilePic = userData?.profilePicUrl ?: post.userProfilePicURL
```

Comment:
What happens when the both condition has blank string ? Shouldn't it better to have a third fallback like 

```suggestion
      val profilePic = userData?.profilePicUrl ?: post.userProfilePicURL ?: ""
```
? 

Let me know what do you think about it

Reply:
Good catch, I will do this, thank youuu!